### PR TITLE
Fix build on windows

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

While doing a bit of cleanup we broke compiling on windows.

Modifications:

Ensure we can also compile on windows.

Result:

Building on windows works again as well.

### CODE REVIEW

**Comment by normanmaurer** [comment]
@nmittler please review as you are guilty =P

File: src/main/c/ssl.c [hunk 1]
```diff
@@ -1228,6 +1228,8 @@ static void ssl_info_callback(const SSL *ssl, int where, int ret) {
 TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
                                                    jlong ctx /* tcn_ssl_ctxt_t * */,
                                                    jboolean server) {
+    SSL *ssl = NULL;
```
Comment (nmittler) [inline src/main/c/ssl.c]: Hmm ... what was wrong with the way it was before?

Comment (normanmaurer) [inline src/main/c/ssl.c]: Stuff needs to be declared on the beginning of the method

Comment (Scottmitch) [inline src/main/c/ssl.c]: C89/ANSI requires declarations to be up front ... didn't' know we were compatible back to c89.

Comment (normanmaurer) [inline src/main/c/ssl.c]: We may be able to fix this in the windows build file... Anyway let us look into it after this was merged

Comment (nmittler) [inline src/main/c/ssl.c]: It's fine ... I don't think we should change the build file.

**Comment by nmittler** [comment]
@normanmaurer  LGTM

**Comment by normanmaurer** [comment]
Cherry-picked into 1.1.33 as 2aa47be27783ec31086ca9881402f845543de4e6

**Pull request closed without merging**